[lit-html] Include SVGElement in render root union type (#5064)
lit/lit: .changeset/sour-heads-study.md

---
'lit-html': patch
---

> Add type SVGElement to render root union type. Change resolves type error when passing an SVGElement as the container argument to render(...), allowing SVG elements to be used as valid render roots.

Reviewer: ```suggestion
Add `SVGElement` to the `render()` container type, allowing SVG elements to be used as render roots without type errors.
```
